cs4360/libs/crm/nest/dao/src/lib/entities/user.entity.ts

```diff
@@ -1,19 +1,19 @@
 import {Column, Entity, JoinColumn, OneToMany, OneToOne, PrimaryGeneratedColumn} from "typeorm";
-import {UserDTO} from "@crm/crm/shared";
+import {UserDTO} from "@crm/shared";
 import {UserTypeEntity} from "./user-type.entity";
 import {ContactEntity} from "./contact.entity";
 
-@Entity('user')
+@Entity('users')
 export class UserEntity implements UserDTO{
 
   @PrimaryGeneratedColumn('uuid')
   id: string
 
-  @Column('varchar', { length: 50, nullable: true})
-  username: string
+  @Column('varchar', {length: 50, nullable: false, unique: true})
+  email: string
```

Comment:
Functional Requirement: Login via email

Reply:
Should we add further validators for the entities at some point?

Reply:
Yeah we totally should. See CS4360-68 Constraints for Entities and Tests to ensure them